src/gax-internal/src/http.rs

```diff
@@ -39,6 +39,7 @@ pub struct ReqwestClient {
     retry_throttler: SharedRetryThrottler,
     polling_error_policy: Arc<dyn PollingErrorPolicy>,
     polling_backoff_policy: Arc<dyn PollingBackoffPolicy>,
+    instrumentation: Option<&'static crate::options::InstrumentationClientInfo>,
```

Comment:
Unless there is a really good reason, I think making a copy here is just fine.  a `ReqwestClient` should be long-lived, and typically wrapped in an `Arc`, so very few copies of it should be taking place.